setup.cfg

```diff
@@ -1,4 +1,4 @@
-[metadata]
+[pimetadata]
```

Comment:
What does this change do?

Reply:
🙈 Typing faster than windows have changed i guess, whoopsie ...